sources/iTermApplicationDelegate.m

```diff
@@ -2063,7 +2063,17 @@ - (IBAction)loadRecording:(id)sender {
 #pragma mark - Private
 
 - (void)updateProcessType {
-    [[iTermApplication sharedApplication] setIsUIElement:[iTermPreferences boolForKey:kPreferenceKeyUIElement]];
+    bool enableLSUI = [iTermPreferences boolForKey:kPreferenceKeyUIElement];
```

Comment:
`BOOL` is how Objective C does it, while `bool` is a c++ism that snuck into a header file somewhere